translations/German/general-notes.md

```diff
@@ -0,0 +1,82 @@
+# General Rules
+
+Thank you for considering to contribute to the German translation!
+Before you start writing, please make sure that you have read the
+following general translation rules.
+
+## Formal or Informal?
+
+**Informal.**
+
+Although written German tends to be quite formal, websites in German are
+usually following informal netiquette. As Zulip's guides are written
+in a more colloquial style, German translations should be rather informal as well.
+
+**Don't use slang or regional phrases in the German translation:**
+* ~~"So'n Dreck kann jedem mal passieren."~~
```

Comment:
I'm not sure we should strike those out, since it's hard to read them - could you rephrase in a more positive way, e.g. "Instead of `<wrong_sentence>` you could say `<right_sentence>`"?